lua/mapvote/sv_mapvote.lua

```diff
@@ -1,227 +1,227 @@
-util.AddNetworkString("RAM_MapVoteStart")
-util.AddNetworkString("RAM_MapVoteUpdate")
-util.AddNetworkString("RAM_MapVoteCancel")
-util.AddNetworkString("RTV_Delay")
+util.AddNetworkString( "RAM_MapVoteStart" )
+util.AddNetworkString( "RAM_MapVoteUpdate" )
+util.AddNetworkString( "RAM_MapVoteCancel" )
+util.AddNetworkString( "RTV_Delay" )
 
 MapVote.Continued = false
 
-net.Receive("RAM_MapVoteUpdate", function(len, ply)
+net.Receive(  "RAM_MapVoteUpdate", function(  _, ply )
     if not MapVote.Allow then return end
-    if not IsValid(ply) then return end
+    if not IsValid( ply ) then return end
 
-    local update_type = net.ReadUInt(3)
+    local update_type = net.ReadUInt( 3 )
     if update_type ~= MapVote.UPDATE_VOTE then return end
 
-    local map_id = net.ReadUInt(32)
+    local map_id = net.ReadUInt( 32 )
     if not MapVote.CurrentMaps[map_id] then return end
-    
+
     MapVote.Votes[ply:SteamID()] = map_id
 
-    net.Start("RAM_MapVoteUpdate")
-        net.WriteUInt(MapVote.UPDATE_VOTE, 3)
-        net.WriteEntity(ply)
-        net.WriteUInt(map_id, 32)
+    net.Start( "RAM_MapVoteUpdate" )
+        net.WriteUInt( MapVote.UPDATE_VOTE, 3 )
+        net.WriteEntity( ply )
+        net.WriteUInt( map_id, 32 )
     net.Broadcast()
-end)
+end )
 
-if file.Exists("mapvote/recentmaps.txt", "DATA") then
-    recentmaps = util.JSONToTable(file.Read("mapvote/recentmaps.txt", "DATA"))
+if file.Exists( "mapvote/recentmaps.txt", "DATA" ) then
+    recentmaps = util.JSONToTable( file.Read( "mapvote/recentmaps.txt", "DATA" ) )
 else
     recentmaps = {}
 end
 
-if file.Exists("mapvote/playcount.txt", "DATA") then 
-    playCount = util.JSONToTable(file.Read("mapvote/playcount.txt", "DATA"))
+if file.Exists( "mapvote/playcount.txt", "DATA" ) then
+    playCount = util.JSONToTable( file.Read( "mapvote/playcount.txt", "DATA" ) )
 else
     playCount = {}
 end
 
 function CoolDownDoStuff()
     cooldownnum = MapVote.Config.MapsBeforeRevote or 3
 
-    if table.getn(recentmaps) == cooldownnum then table.remove(recentmaps) end
+    if #recentmaps == cooldownnum then table.remove( recentmaps ) end
```

Comment:
should double check these are equivalent

Reply:
![image](https://user-images.githubusercontent.com/69946827/189501448-4aa96dcb-51ac-4c05-9f97-52cae1ad72cc.png)